setuptools/__init__.py

```diff
@@ -114,8 +114,10 @@ def setup(**attrs):
 setup.__doc__ = distutils.core.setup.__doc__
 
 if TYPE_CHECKING:
+    from typing_extensions import TypeAlias
+
     # Work around a mypy issue where type[T] can't be used as a base: https://github.com/python/mypy/issues/10962
-    _Command = distutils.core.Command
+    _Command: TypeAlias = distutils.core.Command
```

Comment:
Should we then just do `from distutils.core import Command as _Command`?

Reply:
That works to not use `TypeAlias`, which is more concise, so I think that's an improvement. But you're still stuck with `if TYPE_CHECKING` check.

Reply:
That is OK, just trying to streamline whenever possible...

Reply:
Certainly! And when I have my head deep in it sometimes I'll miss some stuff (like not needing `TYPE_CHECKING` to *use* `TypeAlias`)